tests/std/tests/VSO_0971246_legacy_await_headers/env.lst

```diff
@@ -3,8 +3,8 @@
 
 RUNALL_INCLUDE ..\prefix.lst
 RUNALL_CROSSLIST
-PM_CL="/EHsc /MT /std:c++latest"
-PM_CL="/EHsc /MT /await /std:c++latest"
-PM_CL="/BE /c /EHsc /MD /await /std:c++latest /permissive-"
-PM_CL="/BE /c /EHsc /MD /await /std:c++latest"
-PM_CL="/BE /c /EHsc /MD /std:c++latest"
+PM_CL="/EHsc /MT /std:c++latest /permissive-"
+PM_CL="/EHsc /MT /std:c++latest /permissive"
+PM_CL="/EHsc /MT /std:c++latest /permissive- /await"

```

Comment:
This matrix is special since it actually cares about `/await` (I still don't understand why our *other* tests have `/await` configs, but that's irrelevant here). Should we have both strict and lax configs for `/await` in addition to non-`/await`?

Reply:
IIRC the motivation for having `/await` coverage elsewhere was simply to ensure the compiler didn't break unrelated things with `/await` on the command line - it was cheap. I convinced myself that `/std:c++latest /permissive /await` is an unlikely combination of flags given than both `/permissive` and `/await` are/soon will be non-default, but really I don't care either way. I don't think this test is useful, I don't think we should allow `/await` with `/std:c++latest`, I don't think the cost of more configurations will be significant.

It's easy enough to add another line here, so why not?

Reply:
Thanks - my brain has a heuristic that wants to see a bit more testing for legacy features that we aren't developing or testing elsewhere, since these are precisely the areas that bit-rot in strange configurations.